en/uavcan/notes.md

```diff
@@ -8,7 +8,8 @@ If the PX4 Firmware arms but the motors do not start to rotate, check the parame
 
 ### Debugging with Zubax Babel
 
-A great tool to debug the transmission on the UAVCAN bus is the [Zubax Babel](https://docs.zubax.com/zubax_babel) in combination with the [GUI tool](http://uavcan.org/GUI_Tool/Overview/). They can also be used independently from Pixhawk hardware in order to test a node or manually control UAVCAN enabled ESCs.
+A great tool to debug the transmission on the UAVCAN bus is the [Zubax Babel](https://docs.zubax.com/zubax_babel) in combination with the [GUI tool](http://uavcan.org/GUI_Tool/Overview/).
+They can also be used independently from Pixhawk hardware in order to test a node or manually control UAVCAN enabled ESCs.
 
```

Comment:
@AlexKlimaj This is kind of a troubleshooting page on working with UAVCAN. Is there a good way of describing the symptoms of not having an SD Card? Ie in very first event would it simply be that motors don't turn or is there some other symptom?

If not, how about something like

### UAVCAN devices dont get node ID/FW doesn't update

PX4 requires an SD card for UAVCAN node allocation and firmware upgrade (both of which happen during boot).
Check that there is a (working) SD card present and reboot.

Reply:
All I have to test with are my smart batteries. The symptom is that they don't get a Node ID and the battery information isn't transmitted. So I think what you have is correct.